releasenotes/notes/2.0/64-bit-only-4132f330ec7804b3.yaml

```
@@ -1,16 +1,12 @@
 ---
 upgrade:
   - |
-    Qiskit no longer supports Linux i686 and 32 bit Windows. Starting in Qiskit
-    2.0 a 64 bit platform is needed to run Qiskit. The user base for 32bit
-    architectures is relatively small and as Qiskit continues to focus on
-    improving performance to handle the increased scale in the complexity
-    of quantum computing hardware maintaining support for 32 bit platforms
-    is proving increasingly difficult. This coupled with the larger
-    scientific/numeric Python community's trend away from 32 bit platform
-    support maintaining support for 32bit platforms is no longer viable.
-    Qiskit 1.x will still continue to support 32 bit platforms, but starting
-    in this 2.0.0 release Qiskit no longer supports these platforms, will
-    not publish pre-compiled binaries for these platforms any longer, and
-    there is no longer any guarantee of being able to build from source on
-    32 bit platforms.
+    Qiskit v2.0 has dropped support for Linux i686 and 32-bit Windows. Starting in Qiskit
+    v2.0.0, a 64-bit platform is required to run pre-compiled installations of Qiskit. 
```

I'm not sure when the "pre-compiled" slipped in. But you need a 64 bit platform to run Qiskit >=2.0.0. We make assumptions in the rust code that the pointer width is 64 bits.

```suggestion
    v2.0.0, a 64-bit platform is required to run Qiskit. 
```